.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5470889)